### PR TITLE
reference/sql: update functions and operators

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

update functions and operators in 9 files

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1262/

### Which version does your change affect? 

dev, v3.0

### CODE REVIEW

File: v3.0/reference/sql/functions-and-operators/miscellaneous-functions.md [hunk 1]
```diff
@@ -20,5 +22,14 @@ aliases: ['/docs-cn/sql/miscellaneous-functions/']
 | [`IS_IPV4_COMPAT()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv4-compat)    | 判断参数是否为兼容 IPv4 的地址   |
 | [`IS_IPV4_MAPPED()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv4-mapped)    | 判断参数是否为 IPv4 映射的地址   |
 | [`IS_IPV6()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv6)    | 判断参数是否为 IPv6 地址    |
-| [`GET_LOCK()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_get-lock)    | 获取命名锁，TiDB     出于兼容性支持这个函数，实际上不会做任何操作，这点和 MySQL 有区别 |
-| [`RELEASE_LOCK()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_release-lock)  | 释放命名锁     |
+| [`NAME_CONST()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_name-const)  | 可以用于重命名列名  |
+| [`SLEEP()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_sleep)  | 语句暂停执行几秒时间 |
```
Comment (dcalvin) [inline v3.0/reference/sql/functions-and-operators/miscellaneous-functions.md]:
```suggestion
| [`SLEEP()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_sleep)  | 语句暂停执行的秒数 |
```

Comment (yikeke) [reply v3.0/reference/sql/functions-and-operators/miscellaneous-functions.md]: The second column describes what the function does. So maybe it's better to use "让语句暂停执行几秒时间"?

Comment (dcalvin) [reply v3.0/reference/sql/functions-and-operators/miscellaneous-functions.md]: Right~ my understanding was wrong. Thanks for correction.

**Comment by yikeke** [comment]
Updated. PTAL again @dcalvin

**Pull request merged**